go-ethereum/wait-for-l1.sh

```diff
@@ -46,7 +48,9 @@ if [ ! -z "$DEPLOYER_HTTP" ]; then
 
     exec env \
         ETH1_ADDRESS_RESOLVER_ADDRESS=$ETH1_ADDRESS_RESOLVER_ADDRESS \
+        ETH1_L1_CROSS_DOMAIN_MESSENGER_ADDRESS=$ETH1_L1_CROSS_DOMAIN_MESSENGER_ADDRESS \
         ETH1_NETWORKID=$ETH1_NETWORKID \
+        ROLLUP_ADDRESS_MANAGER_OWNER_ADDRESS=$ROLLUP_ADDRESS_MANAGER_OWNER_ADDRESS \
```

Comment:
Updated to also pass in the manager owner address.

See:
- https://github.com/ethereum-optimism/contracts-v2/blob/c1a9e0fa670e3867e032584568edb60954b04820/bin/deploy.js#L80
- https://github.com/ethereum-optimism/go-ethereum/pull/109/files#diff-7aa83061ca1c69a8ab9203da49a7581bad6e7cc80ecb96582e7fb09c0b8f6534R847